src/main/java/edu/cornell/kfs/fp/document/CuDisbursementVoucherDocument.java

```diff
@@ -618,28 +618,35 @@ public void populateDocumentForRouting() {
         CuDisbursementVoucherPayeeDetail payeeDetail =   getDvPayeeDetail();
 
         if (payeeDetail.isVendor()) {
-            payeeDetail.setDisbVchrPayeeEmployeeCode(getVendorService().isVendorInstitutionEmployee(payeeDetail.getDisbVchrVendorHeaderIdNumberAsInteger()));
-            payeeDetail.setDvPayeeSubjectPaymentCode(getVendorService().isSubjectPaymentVendor(payeeDetail.getDisbVchrVendorHeaderIdNumberAsInteger()));
-        }
-        else if (payeeDetail.isEmployee()|| payeeDetail.isStudent() || payeeDetail.isAlumni()) {
-
-            // Determine if employee student or alumni is a research subject
-            ParameterEvaluator researchPaymentReasonCodeEvaluator = /*REFACTORME*/SpringContext.getBean(ParameterEvaluatorService.class).getParameterEvaluator(DisbursementVoucherDocument.class, DisbursementVoucherConstants.RESEARCH_PAYMENT_REASONS_PARM_NM, payeeDetail.getDisbVchrPaymentReasonCode());
-            if (researchPaymentReasonCodeEvaluator.evaluationSucceeds()) {
-                if (getParameterService().parameterExists(DisbursementVoucherDocument.class, DisbursementVoucherConstants.RESEARCH_NON_VENDOR_PAY_LIMIT_AMOUNT_PARM_NM)) {
-                    String researchPayLimit = getParameterService().getParameterValueAsString(DisbursementVoucherDocument.class, DisbursementVoucherConstants.RESEARCH_NON_VENDOR_PAY_LIMIT_AMOUNT_PARM_NM);
-                    if (StringUtils.isNotBlank(researchPayLimit)) {
-                        KualiDecimal payLimit = new KualiDecimal(researchPayLimit);
-
-                        if (getDisbVchrCheckTotalAmount().isLessThan(payLimit)) {
-                            payeeDetail.setDvPayeeSubjectPaymentCode(true);
-                        }
+            payeeDetail.setDisbVchrPayeeEmployeeCode(
+                getVendorService().isVendorInstitutionEmployee(payeeDetail.getDisbVchrVendorHeaderIdNumberAsInteger()));
+            payeeDetail.setDvPayeeSubjectPaymentCode(
+                getVendorService().isSubjectPaymentVendor(payeeDetail.getDisbVchrVendorHeaderIdNumberAsInteger()));
+        } else if (payeeDetail.isEmployee()) {
```

Comment:
Our older code was also allowing this block to execute if the payee is a student or alumni. We should preserve that customization here.

Reply:
updated